lib/rules/comma-dangle.js

```diff
@@ -81,49 +81,49 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-
         fixable: "code",
-
-        schema: [
-            {
-                defs: {
-                    value: {
-                        enum: [
-                            "always",
-                            "always-multiline",
-                            "only-multiline",
-                            "never"
-                        ]
-                    },
-                    valueWithIgnore: {
-                        anyOf: [
-                            {
-                                $ref: "#/defs/value"
```

Comment:
I haven't looked at this thoroughly yet, but as a quick question: Would this be a breaking change for rules? If some third-party rules have been relying on the (apparently nonstandard) `$ref` property from `is-my-json-valid`, it seems like that won't work anymore if `$ref` is no longer supported in ajv.